src/utils/constants.rs

```diff
@@ -90,6 +90,9 @@ fn agency_map() -> HashMap<&'static str, &'static str> {
     res.insert("eurockot", "Eurockot Launch Services");
     res.insert("rocketlab", "Rocket Lab Ltd");
     res.insert("orbitalatk", "Orbital ATK");
+    res.insert("casc", "China Aerospace Science and Technology Corporation");
+    res.insert("casic", "China Aerospace Science and Industry Corporation");
+    res.insert("pla", "People's Liberation Army");
```

Comment:
```suggestion
    res.insert("cnsa", "China National Space Administration");
```

Reply:
pla is not an agency, but the cnsa is the national agency, so I assume you meant that one 😅

Reply:
D'oh, I was looking at the 1.4.1 API and not the 2.0